content/en/docs/concepts/configuration/secret.md

```diff
@@ -819,74 +529,16 @@ field set to that of the service account.
 See [Add ImagePullSecrets to a service account](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account)
```

Comment:
The link to "Add ImagePullSecrets to a service account" is repeated on about 20 lines up from here. Just want to make sure you meant to have it in both places.